File: src/main-service/Doerly.Module.Order/Doerly.Module.Order.DataTransferObjects/Responses/Feedback/OrderFeedbackResponse.cs
namespace Doerly.Module.Order.DataTransferObjects.Responses;

public class OrderFeedbackResponse
{
    public int FeedbackId { get; set; }
    
    public int Rating { get; set; }

    public string? Comment { get; set; }

    public ProfileInfo UserProfile { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: src/doerly-fe/src/app/modules/order/models/responses/feedback/order-feedback-response.ts
import {ProfileInfoModel} from 'app/modules/order/models/responses/profile-info-model';

export interface OrderFeedbackResponse {
  feedbackId: number;
  rating: number;
  comment: string;
  userProfile: ProfileInfoModel;
  createdAt: Date;
  updatedAt: Date;
}
